### ISSUE TITLE
Plot confidence area around a linegraph

### ISSUE THREAD

**Issue by fgvieira** [MultiQC/MultiQC#2214]
### Description of feature

Is it possible to plot confidence areas (e.g. confidence intervals or standard deviations) around a line graph?

Like [this](https://stackoverflow.com/a/32616695) example.

**Comment by ewels** [comment]
No, and I have avoided doing this in the past because it does not scale well. Having a single additional band as suggested in https://github.com/ewels/MultiQC/issues/2218 is fine, but having a confidence area per-sample is bad as the graph will quickly become unreadable when there are more than a few samples.

Closing as a duplicate of https://github.com/ewels/MultiQC/issues/2218

**Comment by vladsavelyev** [comment]
I wonder if we can still do that only if the number of samples is low. It would fit well into the "plot scaling" effort. 

E.g. a confidence area can be displayed _only_ if there is only a single sample. Some additional per-sample dots can be displayed only if there are _few_ samples. When scaling further, we will omit some lines and fall back for different representations, like discussed [here](https://github.com/MultiQC/MultiQC/issues/2280). 

Certain modules are often run on a single sample anyway, so the problem of scaling is not that important there.
